config/samples/ovs_v1beta1_ovs.yaml

```diff
@@ -10,5 +10,7 @@ spec:
     ovn-bridge: "br-int"
     ovn-encap-type: "geneve,vxlan"
   nic_mappings:
-    physnet1: enp2s0.100
-    physnet2: enp2s0.200
+    physnet2: enp2s0.100
+    physnet3: enp2s0.200
+  tunnelNetworkCidr: 10.20.30.40/24
+  tunnelNetworkNic: enp2s0.300
```

Comment:
Not sure I understand exactly how that works. IIUC this mean that all nodes where the daemonset creates the pod will have the same IP active at the same time? 
As part of the isolated network topic I am extracting/rework CRDs from OSPdO which will use node network configuration policies (nncp) to configure e.g. a bridge interfaces on the worker nodes. This then will be used in NAD for attaching interface to the pod.
Is this tunnelNetworkNic special, or should be one of those, like in traditional ooo an internalapi network interface?

Reply:
It's not special at all. TBH I was working on it only on CRC machine and I was wondering how it will be on "real" cluster with more than one node. If this is done in wrong way and we should use that "nncp" please just point me to some guideline how to do it :)